Cargo.toml

```diff
@@ -343,6 +349,8 @@ wasm = ["lucet-runtime", "lucet-wasi", "lucetc", "vector-wasm"]
 # transforms and sinks should depend on this feature.
 kubernetes = ["k8s-openapi", "evmap"]
 
+docker = ["bollard", "dirs-next"]
```

Comment:
New feature to extract some of the bits from the `docker_logs` source to share with the `fluent` integration tests.